src/netlog/netlog-manager.c

```diff
@@ -113,6 +114,18 @@ static int manager_read_journal_input(Manager *m) {
                 r = parse_field(data, length, "MESSAGE=", &message);
                 if (r < 0)
                         return r;
+                else if (r > 0)
+                        continue;
+
+                r = parse_field(data, length, "SYSLOG_STRUCTURED_DATA=", &structured_data);
```

Comment:
Unlike `SYSLOG_IDENTIFIER`, `SYSLOG_STRUCTURED_DATA` and `SYSLOG_MSGID` are not defined by Journal. 
(See fields defined by journal here: https://manpages.debian.org/bullseye/systemd/systemd.journal-fields.7.en.html#USER_JOURNAL_FIELDS)
These are names that I came up with.
I was thinking about making these names configurable in case some application already uses such field names for something else, but that is unlikely. I expect that these field names are not yet in use by applications that expect systemd-netlogd to forward their journal entries.